src/app/components/carousel/carousel.component.html

```diff
@@ -1,4 +1,20 @@
-<ion-icon name="chevron-back-outline" size="large" class="cursor-pointer"></ion-icon>
-<ion-icon name="chevron-forward-outline" size="large" class="cursor-pointer"></ion-icon>
-<img src="../../../assets/black-friday-elements-assortment_23-2149074076.avif" id="image"/>
+<div id="slider">
+    <!-- <ion-icon name="chevron-back-outline" size="large" class="cursor-pointer"></ion-icon>
+    <ion-icon name="chevron-forward-outline" size="large" class="cursor-pointer"></ion-icon> -->
+    
+    <img id="slider-1" src ="../../../assets/black-friday-elements-assortment_23-2149074076.avif" class="image"/>
+    <img id="slider-2" src="../../../assets/images.jpg" class="image"/>
```

Comment:
find a higher quality image

Reply:
There seems to be a red border here
![image](https://github.com/stevehotcodes/angular_ecommerce_frontend/assets/24773276/d8a687ea-ad52-4675-b6f9-33e2f367faac)